src/Dock.Avalonia.Themes.Default/DockControl.xaml

```diff
@@ -41,35 +41,12 @@
                             </ItemsControl>
                         </DataTemplate>
                         <DataTemplate DataType="Controls:IPinDock">
-                            <DockPanel Name="DropPin" id:DockProperties.IsDropArea="True" Background="Transparent">
-                                <TabStrip Items="{Binding VisibleDockables}" SelectedItem="{Binding ActiveDockable, Mode=TwoWay}" Focusable="false" ClipToBounds="False" DockPanel.Dock="{Binding Alignment, Converter={StaticResource AlignmentConverter}}">
-                                    <TabStrip.Styles>
-                                        <Style Selector="TabStrip:empty">
-                                            <Setter Property="IsVisible" Value="False" />
-                                        </Style>
-                                        <Style Selector="TabStrip /template/ ItemsPresenter#PART_ItemsPresenter > WrapPanel">
-                                            <Setter Property="Orientation" Value="Vertical"/>
-                                        </Style>
-                                    </TabStrip.Styles>
-                                    <TabStrip.DataTemplates>
-                                        <DataTemplate DataType="Dock:IDockable">
-                                            <StackPanel Name="DragPin" id:DockProperties.IsDragArea="True" id:DockProperties.IsDropArea="True" Background="Transparent" Orientation="Horizontal" Spacing="2">
-                                                <TextBlock Text="{Binding Title}" Classes="drag" Margin="2" />
-                                            </StackPanel>
-                                        </DataTemplate>
-                                    </TabStrip.DataTemplates>
-                                </TabStrip>
-                                <ContentControl Content="{Binding ActiveDockable}" IsVisible="{Binding IsExpanded}"/>
-                            </DockPanel>
+                            <idc:PinDockControl/>
```

Comment:
I've initially tried to do everything in PinDockControl, so moved it to external file, and after introduction of PinHostControl left it in separate file, I think it's better to keep it separate file as it's not that simple.